Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -222,12 +225,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 510DCB3F1CCA69ED0075E8CB /* Build configuration list for PBXNativeTarget "EmissionTests" */;
 			buildPhases = (
+				20B6FFACE9798C3F6AEDEF21 /* [CP] Check Pods Manifest.lock */,
```

Comment:
they shouldn't be any. But I have to pod install to get it to run, and when I do so, there are project and podfile.lock changes